bastion/examples/parallel_computation.rs

```diff
@@ -58,7 +58,7 @@ fn main() {
                 for id_worker_pair in workers.elems().iter().enumerate() {
                     let data = cycle(id_worker_pair.0 as u64, 5);
 
-                    let computed: Answer = id_worker_pair.1.ask(data).unwrap();
+                    let computed: Answer = ctx.ask(&id_worker_pair.1.addr(), data).unwrap();
```

Comment:
Are we sure that we want a tuple variant at the public API? I don't think that it is a good option tbh.